scripts/main.js

```diff
@@ -0,0 +1,83 @@
+const btnSubmit = document.querySelector('.submit');
+
+let bookObj;
+
+let bookid = 0;
+let author = ' ';
+let title = ' ';
+let bookListObj = [];
+let deserializedBookList;
+
+function saveData(dataObj) {
+  const dataString = JSON.stringify(dataObj);
+  localStorage.setItem('bookList', dataString);
+}
+
+function getData(key) {
+  return JSON.parse(localStorage.getItem(key));
+}
+
+function getBookList() {
+  let books = ' ';
+  deserializedBookList = getData('bookList');
+
+  deserializedBookList.forEach((value) => {
+    books += `
+            <div class="book" data-id="${value.bookid}">
+              <h2 class="title">${value.title}
+                <span class="author">${value.author}</span>
+              </h2>
+              <button class="btn btn-remove" type="button">Remove</button>
+              <hr/>
+            </div>
+              `;
+  });
+
+  const booklist = document.querySelector('.book-display');
+  booklist.innerHTML = books;
+}
+
+function addToList(title, author) {
+  bookListObj = getData('bookList');
+
+  if (bookListObj.length > 0) {
```

Comment:
When I click on the `Add` button I am getting this error.

<img width="1440" alt="Screen Shot 2022-06-28 at 8 00 25" src="https://user-images.githubusercontent.com/82426682/176184727-75a0a1e5-5102-41a6-91ac-3dfdfeac92b3.png">

Please fix it.

Reply:
@vechicin Thank you for your review, can you please verify again? We are not facing this issue.